### DISCUSSION TITLE
Can I .gitignore the `artifacts` folder? It has enormous files in it.

### DISCUSSION THREAD

**Discussion by AJTJ** [react-static/react-static#1495] [General] [answered]
As the title suggests.

I'm using NetlifyCMS and will have github actions automatically building/compiling the project for deployment through github pages. I'm wondering if there will be some sort of conflict.

**Comment by SleeplessByte** [comment, marked as answer]
You can safely gitignore any build folder, including the artifacts folder. It is, by default, ignored in the [basic template](https://github.com/react-static/react-static/blob/master/packages/react-static/templates/basic/.gitignore), but some templates lack it.

**Reply by AJTJ** [reply]
thanks!
